demo/src/app/components/shared/api-docs/api-docs-config.component.html

```diff
@@ -1,22 +1,37 @@
 <div (click)="trackSourceClick()" class="api-doc-component">
-  <h2>
-    <a [routerLink]="" fragment="{{apiDocs.className}}" ngbdFragment>
+  <h3>
+    <a
+      class="title-fragment"
+      [routerLink]=""
+      fragment="{{apiDocs.className}}"
+      ngbdFragment
+      title="Anchor link to: {{apiDocs.className}}"
+    >
       <img src="img/link-symbol.svg" alt="Anchor link to: {{apiDocs.className}}"/>
     </a>
-    <a href="https://github.com/ng-bootstrap/ng-bootstrap/tree/master/{{apiDocs.fileName}}" target="_blank">{{apiDocs.className}}</a>
-  </h2>
-  <p>{{apiDocs.description}}</p>
+    {{apiDocs.className}}
+    <a
+      class="github-link"
+      href="https://github.com/ng-bootstrap/ng-bootstrap/tree/master/{{apiDocs.fileName}}"
+      target="_blank"
+      title="Link to Github: {{apiDocs.className}}"
+    >
+      <img src="img/github.svg" alt="Link to Github {{apiDocs.className}}"/>
+    </a>
+  </h3>
+  <h5><small class="text-muted">Service Configuration</small></h5>
```

Comment:
Configuration Service?

Reply:
Or just 'Configuration'? It is not really a service, just an injectable object.

Reply:
I'll update it to just _Configuration_